bin/ogit.ml

```diff
@@ -177,6 +178,63 @@ let cat = {
       end in
     Term.(mk cat_file $ file)
 }
+*)
```

Comment:
I personally find that command useful, so I'd like to keep if possible. So maybe simply rename it to `git cat`?